tests/ttnn/unit_tests/operations/test_all_reduce_t3000_frequent.py

```diff
@@ -18,6 +18,8 @@ def is_unsupported_case(input_shape, math_op, mem_config, num_devices, num_links
     num_l1_banks = 64
     if mem_config.buffer_type == ttnn.BufferType.L1 and tensor_size_bytes > num_l1_banks * 50 * 1024:
         return True, "L1 buffer can't support large tensor sizes"
+    if input_shape[3] == 32 and input_dtype == ttnn.bfloat8_b:
```

Comment:
@SeanNijjar the reason for this skip is that the shape with h or w as 32 is being hang, so we can redirect it to unoptimized flow but the problem there is reshape op is calling tilize op internally and that supported only bf16 alone it seems, what can we do in that case?

Reply:
I see - this was an existing issue with the existing implementation? It's a bit insane that reshape does tilize on outer-dim reshapes since those are always reinterprets.

This is something we'll hit during generality push. I'm okay with your change here as is based on what you've told me.